ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -83,6 +83,10 @@ object LedgerApiTestTool {
   def main(args: Array[String]): Unit = {
 
     val config = Cli.parse(args).getOrElse(sys.exit(1))
+    println(config)
+    if (Math.random() < 1) {
+      sys.exit(0)
+    }
```

Comment:
Clever. Now remove it. 😂

Reply:
How did I manage to leave this in? :joy: